osu.Game.Rulesets.Catch/Edit/CatchHitObjectComposer.cs

```diff
@@ -42,6 +42,10 @@ public CatchHitObjectComposer(CatchRuleset ruleset)
         [BackgroundDependencyLoader]
         private void load()
         {
+            // todo: enable distance spacing once catch supports it.
```

Comment:
May want to reword this comment to mention that the part which isn't supported by the osu!catch editor isn't distance spacing itself, but the ability to adjust the applied amount.